lua/core/default_config.lua

```diff
@@ -148,7 +152,7 @@ M.mappings.plugin = {
       next_buffer = "<TAB>", -- next buffer
       prev_buffer = "<S-Tab>", -- previous buffer
    },
-   chadsheet = {
```

Comment:
nooooooooooo :sob: :laughing:

Reply:
Yes :chad :

It's confusing, I think it's funny, but too confusing

Soz dude